apps/api/src/server/applications/applications.routes.js

```diff
@@ -139,4 +142,33 @@ router.get(
 	asyncHandler(getApplicationDetails)
 );
 
+router.patch(
+	'/:caseId/documents/:documentGUID/status',
+	/*
+        #swagger.tags = ['Applications']
+        #swagger.path = '/:caseId/documents/:documentGUID/status'
```

Comment:
Is this the correct path?

Reply:
corrected it to: 'applications/{caseId}/documents/{documentGUID}/status'